sdk/communication/Azure.Communication.Chat/src/Generated/ChatThreadRestClient.cs

```diff
@@ -753,40 +754,49 @@ public Response<ChatParticipantsCollection> ListChatParticipants(string chatThre
             }
         }
 
-        internal HttpMessage CreateRemoveChatParticipantRequest(string chatThreadId, string chatParticipantId)
+        internal HttpMessage CreateRemoveChatParticipantRequest(string chatThreadId, string rawId, CommunicationUserIdentifierModel communicationUser, PhoneNumberIdentifierModel phoneNumber, MicrosoftTeamsUserIdentifierModel microsoftTeamsUser)
         {
             var message = _pipeline.CreateMessage();
             var request = message.Request;
-            request.Method = RequestMethod.Delete;
+            request.Method = RequestMethod.Post;
             var uri = new RawRequestUriBuilder();
             uri.AppendRaw(endpoint, false);
             uri.AppendPath("/chat/threads/", false);
             uri.AppendPath(chatThreadId, true);
-            uri.AppendPath("/participants/", false);
-            uri.AppendPath(chatParticipantId, true);
+            uri.AppendPath("/participants/:remove", false);
             uri.AppendQuery("api-version", apiVersion, true);
             request.Uri = uri;
             request.Headers.Add("Accept", "application/json");
+            request.Headers.Add("Content-Type", "application/json");
+            var model = new CommunicationIdentifierModel()
```

Comment:
We are not doing this manually here, right ?

Reply:
no, this is auto generated